internal/integration/sqlite_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"ariga.io/atlas/sql/schema"
 	"ariga.io/atlas/sql/sqlite"
 
+	_ "github.com/libsql/libsql-client-go/libsql"
```

Comment:
Hey! Why these changes are necessary? We agreed the tests will reside on a separate repo.

Reply:
I'm sorry - I must have misunderstood you. I thought that only libsql specific tests should live in separate repo. This PR makes it possible to run existing Atlas sqlite tests against libsql. It does not change any tests* - just adds ability to select different target. Default target is still sqlite and no one will experience any difference in tests behaviour unless they explicitly set the right env variables. I was hoping you would agree to merge this to master together with libsql support. That would help us avoid merge conflicts in case you decide to change sqlite tests in the future.

As we agreed, I created a separate repo that runs the tests once a day -> https://github.com/chiselstrike/atlas-integration-ci
This was my understanding of the separate repo idea. We (Libsql maintainers) maintain and monitor the integration repo to make sure integration keeps working.

There are too alternatives to what I did:
1. I copy all sqlite tests to the integration repo but then it creates unnecessary maintenance cost of keeping them synced with upstream
2. In integration repo, I could apply my patch to the current atlas master. This on the other hand will break when Atlas sqlite tests are changed

Would it be ok to merge this @a8m ? 

* - I had to restructure one test but functionally it's the same as before. I just had to wrap the code into `t.Run`

Reply:
Why testing libsql is needed here? Atlas does not have any special handling besides opening a connection. i.e., the validation in the other repository is enough IMO.

Sorry for the misunderstanding and for being too picky here, we use the same (package) tests for our internal testing (and commercial drivers) and I prefer to not have it there. 

> I had to restructure one test but functionally it's the same as before. I just had to wrap the code into t.Run

Can you elaborate on why it's needed?